srcpkgs/quakespasm/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'quakespasm'
+pkgname=quakespasm
+version=0.93.1
+revision=1
+build_style=gnu-makefile
+build_wrksrc="Quake"
+makedepends="SDL2-devel libmad-devel"
+make_build_args="DO_USERDIRS=1 USE_SDL2=1"
+make_use_env=yes
+short_desc="Modern Quake 1 engine"
+maintainer="SolitudeSF <solitudesf@protonmail.com>"
+license="GPL-2.0-or-later"
+homepage="http://quakespasm.sourceforge.net/"
+distfiles="${SOURCEFORGE_SITE}/quakespasm/Source/quakespasm-${version}.tgz"
+checksum=ed3ee6ba7e3d9454c7620757cc166e13d892f1167c7b74bef4f067e350d835ae
+
+case "$XBPS_TARGET_MACHINE" in
+	arm*) makedepends+=" libGL"
```

Comment:
i dislike this "hack" but i dont know if there is a cleaner and easier solition.

Reply:
I don't know, I trust you. :neutral_face: